CONTRIBUTING.md

```diff
@@ -0,0 +1,164 @@
+# Contribute to Atlas documentation
+
+Atlas allows for ANYONE to edit and create documentation for Node Engine, Faust.js, and Content Engine. Notably, we love it when users:
```

Comment:
Add Content Modeler here as there is a PR in-flight, or simply mention "all products"